rma/models/rma.py

```diff
@@ -0,0 +1,715 @@
+# Copyright 2020 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+import odoo.addons.decimal_precision as dp
+
+RMA_OPERATIONS = [
+    ("refund", "Refund"),
+    ("replace", "Replace"),
+    ("repair", "Repair")
+]
+
+
+class Rma(models.Model):
+    _name = "rma"
+    _description = "RMA"
+    _order = "priority,date desc"
+    _inherit = ["mail.thread", "portal.mixin"]
+
+    # General fields
+    sent = fields.Boolean()
+    name = fields.Char(
+        string='Name',
+        index=True,
+        readonly=True,
+        states={'draft': [('readonly', False)]},
+        copy=False,
+        default=lambda self: _('New'),
+    )
+    origin = fields.Char(
+        string='Source Document',
+        help="Reference of the document that generated this RMA.",
+    )
+    date = fields.Datetime(
+        default=lambda self: fields.Datetime.now(),
+        copy=False,
+        Index=True,
+        required=True,
+        readonly=True,
+        states={'draft': [('readonly', False)]},
+    )
+    deadline = fields.Date(
+        copy=False,
+    )
+    date_closed = fields.Datetime(
+        string="Closed",
+        copy=False,
+    )
+    user_id = fields.Many2one(
+        comodel_name="res.users",
+        string="Responsible",
+        track_visibility="always",
+        default=lambda self: self.env.user,
+        readonly=True,
+        states={'draft': [('readonly', False)]},
+    )
+    team_id = fields.Many2one(
+        comodel_name="rma.team",
+        string="RMA team",
+        index=True,
+        help="Responsible RMA team."
+             " Define Responsible user and Email account for"
+             " mail gateway.",
+    )
```

Comment:
Subscribing users to the team doesn't subscribe them to the team's RMA's. I think it should work the same way other models do (crm.teams, projects...)

Reply:
Still not working